python/masonite/app/http/middleware/AuthenticationMiddleware.py

```diff
@@ -0,0 +1,24 @@
+"""Authentication Middleware."""
```

Comment:
Are **middlewares** required for this setup ?

Reply:
Not really, i can remove them.